Server/JokesOnYou.Web.Api/DTOs/JokeWithAuthorReplyDto.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace JokesOnYou.Web.Api.DTOs
+{
+    public class JokeWithAuthorReplyDto : JokeReplyDto
```

Comment:
a dto Inhereting from another dto is generally not a good idea, since they should look pretty different otherwise we can reuse the same